test-framework/security-jwt/src/main/java/io/quarkus/test/security/jwt/Type.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.test.security.jwt;
+
+import java.io.StringReader;
+
+import jakarta.json.Json;
+import jakarta.json.JsonArray;
+import jakarta.json.JsonObject;
+import jakarta.json.JsonReader;
+
+public enum Type {
```

Comment:
Looks good but now that it's an external structure, shouldn't we name it something like `ClaimType` instead of just `Type`?

Reply:
@gastaldi I thought making it external allows for some micro optimization with `@Claim(key="custom", value="567", type=Type.LONG)` is a few characters shorter :-) than `@Claim(key="custom", value="567", type=ClaimType.LONG)`, but may be it is a bit silly :-), are you concerned of naming the same as the Java `Type` class ? I can rename it for sure

Reply:
@gastaldi Or just `fix` it by adding JavaDocs to `Type` explaining it is a claim type :-) ?

Reply:
@gastaldi Will rename to ClaimType as it is indeed better

Reply:
Done, and squashed a few commits too, thanks